src/pages/Details.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { BiArrowBack } from 'react-icons/bi';
+import './styles/Details.css';
+
+function Details() {
+  const { name } = useParams();
+  const { data } = useSelector((state) => state.country);
+  const countryDetails = data.find((country) => country.name === name);
+  const navigation = useNavigate();
+
+  if (!countryDetails) {
+    return (
+      <div className="not-found">
+        <h2>Country Not Found</h2>
+        <button
+          type="button"
+          onClick={() => navigation('/')}
+          className="back-btn"
+        >
+          <BiArrowBack />
+        </button>
+      </div>
+    );
+  }
+
+  return (
+    <div className="details-container">
+      <div className="details-country">
+        <div className="background">
+          <button
+            type="button"
+            onClick={() => navigation('/')}
+            className="back-btn"
+          >
+            <BiArrowBack />
+          </button>
+        </div>
+        <h2 className="header">
+          {countryDetails.name}
+          &apos;s Details
+        </h2>
+      </div>
+      <div className="container">
+        <img src={countryDetails.flag} alt={countryDetails.name} />
+        <div className="flex-container">
+          <h2 className="country-name-details">{countryDetails.name}</h2>
+          <h3 className="country-details-off">
+            Official Name:
+            {' '}
+            {countryDetails.officialName}
+          </h3>
+          <h4 className="country-details">
+            Capital City:
+            {' '}
+            {countryDetails.capital}
+          </h4>
+          <h4 className="country-details">
+            Continent:
+            {' '}
+            {countryDetails.continent}
+          </h4>
+          <h4 className="country-details">
+            Population:
+            {' '}
+            {countryDetails.population}
+          </h4>
+          <h4 className="country-details">
+            Area:
+            {countryDetails.area}
+          </h4>
+          <h4 className="country-details">
+            Timezone:
+            {countryDetails.timezone[0]}
+          </h4>
+        </div>
+      </div>
```

Comment:
- [ ] The details page for each country is not displaying as nicely as expected on mobile screens. Kindly work on displaying content for the details page to match your design with the original design. 👍🏽

<img width="478" alt="Screenshot 2023-10-11 at 10 27 16 PM" src="https://github.com/siddghosh108/React-Country-Capstone/assets/50941074/c08a73e3-bfdf-4c93-8ef5-c8f38097c5cf">

Reply:
Please see the original design given below:
![image](https://github.com/siddghosh108/React-Country-Capstone/assets/131663359/9ebedb65-d3c3-4b5b-8559-33e9924ad597)
 My design of the detail page is exactly like the original design so what do you mean by changing it is not understood from my end. Please approve the project as i have completed alll the design changes asked for